bin/lib/locale-compare.js

```diff
@@ -6,6 +6,7 @@ var crypto = require('crypto');
 var fs = require('fs');
 var path = require('path');
 var po2icu = require('po2icu');
+var isArray = require('lodash.isArray');
```

Comment:
No, actually I didn't realize there were tests for the script. The new md5 map expects the value to be an array not a string, but I hadn't changed the fixture.